modules/ROOT/pages/_partials/push/8_handling-push-notifications.adoc

```diff
@@ -40,15 +43,15 @@ iOS::
 +
 --
 
-The {SDK} for iOS does not provide a way to handle incoming messages. Instead, follow the link:https://developer.apple.com/notifications/[Apple's official documentation^] about notifications.
+The {SDK} for iOS does not provide a way to handle incoming messages. Instead, follow the link:https://developer.apple.com/notifications/[Apple's official documentation, window="_blank"] about notifications.
 
 --
 // end::excludeDownstream[]
 Cordova::
 +
 --
 
-The {SDK} for JavaScript does not provide a way to handle incoming messages. We recommend using the link:https://ionicframework.com/docs/native/push[Ionic Push plugin^].
+The {SDK} for JavaScript does not provide a way to handle incoming messages. We recommend using the link:https://ionicframework.com/docs/native/push[Ionic Push plugin, window="_blank"].
```

Comment:
window="_blank" won't work in downstream

Reply:
@finp - @pmacko1 : when I look at this merged [PR ](https://github.com/aerogear/mobile-docs/commit/8e614ac3cbd3552a7e4561057e804102d606d3ee) - I become confused by this syntax. @finp - what is the right syntax to use here?

Reply:
link:url[text^]